src/software/ai/hl/stp/stp_test.cpp

```diff
@@ -84,10 +90,10 @@ TEST_F(STPTest, test_calculate_new_play_when_multiple_plays_valid)
     }
 
     std::vector<std::string> expected_play_names = {
-        TYPENAME(MoveTestPlay), TYPENAME(MoveTestPlay), TYPENAME(MoveTestPlay),
-        TYPENAME(MoveTestPlay), TYPENAME(MoveTestPlay), TYPENAME(MoveTestPlay),
-        TYPENAME(MoveTestPlay), TYPENAME(MoveTestPlay), TYPENAME(HaltTestPlay),
-        TYPENAME(MoveTestPlay),
+        TYPENAME(HaltTestPlay), TYPENAME(HaltTestPlay), TYPENAME(MoveTestPlay),
+        TYPENAME(MoveTestPlay), TYPENAME(HaltTestPlay), TYPENAME(MoveTestPlay),
+        TYPENAME(HaltTestPlay), TYPENAME(MoveTestPlay), TYPENAME(HaltTestPlay),
+        TYPENAME(HaltTestPlay),
```

Comment:
the tactic assignment returns plays in this order now because of some changes to the play registry

the rng is initialized with a seed so this test is repeatable